### DISCUSSION TITLE
how to update a json field's member?

### DISCUSSION THREAD

**Discussion by yunfan** [prisma/prisma#8363] [Q&A] [answered]
i havnt found the relevent issue here and the document from official

suppose i want to do operation like 
```
update staff set data = json_set(data,"$.gender","male") where id = 1;
```

how should i do this using prisma?

**Comment by ryands17** [comment, marked as answer]
@yunfan 👋 

Currently there's no direct way to update a Json field's member. You would need to fetch the data first, update the specific field and then call `update`  with the updated value.

It would be great if you could add a 👍 on this request so that we can look at the priority for this: https://github.com/prisma/prisma/issues/5057

**Reply by yunfan** [reply]
thanks

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
